container/container.go

```diff
@@ -2260,3 +2283,30 @@ func isAnaxNetwork(net *docker.Network, bridgeName string) bool {
 	}
 	return false
 }
+
+// Verify that the permission bits for the host side of the binding allow anyone/other
+// to access that file or directory.
+func hasValidBindPermissions(binds []string) error {
+	for _, bind := range binds {
+		containerVol := strings.Split(bind, ":")
+		if len(containerVol) > 1 && containerVol[1] != "" {
```

Comment:
We should cover the case where there is only 1 element. like "/mydir". This means both host and docker container has the same dir name.

Reply:
ok i wondering about that case.